### PR TITLE
Update logstash to latest version

### PR DESCRIPTION
**Pull request by d34dh0r53**
Updated logstash to latest version for mitaka work.

Connects rcbops/u-suk-dev#176

### CODE REVIEW

File: rpcd/playbooks/roles/logstash/tasks/logstash_post_install.yml [hunk 1]
```diff
@@ -24,13 +24,14 @@
...
-    state: stopped
-    enabled: no
+- name: Create patterns directory
+  file:
+    name: "/opt/logstash/patterns"
+    owner: "logstash"
+    group: "logstash"
+    state: directory
   tags:
-    - logstash-web-disable
```
Comment (sigmavirus24) [inline rpcd/playbooks/roles/logstash/tasks/logstash_post_install.yml]: Why the change to the tags?

**Comment by sigmavirus24** [comment]
I have one question, but this looks fine to me.

File: rpcd/playbooks/roles/logstash/tasks/logstash_post_install.yml (hunk 1 above)
Comment (d34dh0r53) [inline rpcd/playbooks/roles/logstash/tasks/logstash_post_install.yml]: That's actually because the `Create patterns directory` is in the same place and is the same number of lines as the defunct and removed `Logstash web disable` task.

**Comment by prometheanfire** [comment]
looks ok to me

:+1:

**Comment by sigmavirus24** [comment]
@d34dh0r53 is this actually a WIP or can we merge it?

**Comment by d34dh0r53** [comment]
@sigmavirus24 this is currently a WIP as the service doesn't start due to config issues.

**Comment by andymcc** [comment]
Is this still a WIP?

**Comment by Apsu** [comment]
RPC-AIO-Liberty-PR: Fail

**Comment by d34dh0r53** [comment]
recheck_swift

**Comment by Apsu** [comment]
RPC-AIO-Liberty-PR: Fail

**Comment by Apsu** [comment]
RPC-AIO-Ceph-Liberty-PR: Fail

**Comment by prometheanfire** [comment]
:+1:

**Comment by mancdaz** [comment]
Has anyone actually tested this, or just tested with their eyes?

**Comment by sigmavirus24** [comment]
So I have an AIO with this, #1245 and #1246 running and it seems to be working. Logstash is shipping things to elasticsearch and kibana is accessible and showing the items in queries out of elasticsearch.

So a 👍 after having tested this.

**Comment by andymcc** [comment]
👍  I replicated what @sigmavirus24 saw, and was able to get a working ELK stack, in that logs are being passed all the way through to elasticsearch and are visible by kibana.
